shanoir-ng-front/src/app/import/pre-clinical-context/pre-clinical-context.component.ts

```diff
@@ -87,6 +87,7 @@ export class PreClinicalContextComponent extends AbstractClinicalContextComponen
         importJob.acquisitionEquipmentId = context.acquisitionEquipment.id;
         importJob.converterId = context.niftiConverter.id;
         importJob.archive = contextImportJob.archive;
+        importJob.timestamp = timestamp;

```

Comment:
Maybe initialize timestamp here ? (same for basic-clinical-context.component.ts, maybe code refactoring is possible ?)